tutorials/multiple-fields/left.py

```diff
@@ -0,0 +1,64 @@
+from mpi4py import MPI
+from dolfinx import mesh
+from dolfinx.fem import functionspace
+from dolfinx import fem
+import numpy
+import ufl
+from dolfinx import default_scalar_type
+from dolfinx.fem.petsc import LinearProblem
+
+import fenicsxprecice
+
+domain = mesh.create_rectangle(
+    MPI.COMM_WORLD, [
+        numpy.asarray(
+            (0, 0)), numpy.asarray(
+                (1, 1))], [
+                    10, 10], mesh.CellType.quadrilateral)
+domain2 = mesh.create_rectangle(
+    MPI.COMM_WORLD, [
+        numpy.asarray(
+            (0, 0)), numpy.asarray(
+                (1, 1))], [
+                    11, 11], mesh.CellType.quadrilateral)
+V = functionspace(domain, ("Lagrange", 1))
```

Comment:
```suggestion
V1 = functionspace(domain, ("Lagrange", 1))
```
Because you are talking about `V2` below